app/javascript/components/visualization/Heatmap.js

```diff
@@ -81,11 +75,15 @@ export default function Heatmap({
   useUpdateEffect(() => {
     if (morpheusHeatmap.current && morpheusHeatmap.current.fitToWindow) {
       const fit = heatmapFit
-      morpheusHeatmap.current.fitToWindow({
-        fitRows: fit === 'rows' || fit === 'both',
-        fitColumns: fit === 'cols' || fit === 'both',
-        repaint: true
-      })
+      if (fit === '') {
+        morpheusHeatmap.current.resetZoom()
```

Comment:
nice!  I probably spent 30mins digging through the morpheus code looking for something like this resetZoom function before I gave up

Reply:
Yeah it took a while to find it, and I only did by clicking through the toolbar.